### PR TITLE
update static root settings

### PR DESCRIPTION
**Pull request by moh-moola**

### CODE REVIEW

File: project/settings.py [hunk 1]
```diff
@@ -404,7 +404,7 @@
     )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env.bool("DEBUG", False)
+DEBUG = env.bool("DEBUG", True)
```
Comment (codiebeulaine) [inline project/settings.py]: ?

Comment (moh-moola) [reply project/settings.py]: By default the Debug mode should be set to True and explicitly set to False in production, but this is not the case for the Auth Service in prod

**Comment by moh-moola** [comment]
@codiebeulaine updated the above issue

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**